homeassistant/components/apple_tv/manifest.json

```diff
@@ -3,8 +3,15 @@
   "name": "Apple TV",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/apple_tv",
-  "requirements": ["pyatv==0.8.2"],
-  "zeroconf": ["_mediaremotetv._tcp.local.", "_touch-able._tcp.local."],
+  "requirements": ["pyatv==0.9.6"],
+  "zeroconf": [
+    "_mediaremotetv._tcp.local.",
+    "_touch-able._tcp.local.",
+    "_appletv-v2._tcp.local.",
+    "_airplay._tcp.local.",
+    "_raop._tcp.local.",
+    "_hscp._tcp.local."
```

Comment:
Do we need the new ids for this PR?

Reply:
Separate PRs for each service?

Reply:
No, just one PR that adds new services.

This PR is about upgrading to a new library version and making only the changes needed to still have a working integration. All other changes should go in other PRs to keep the scope as small as possible.

Reply:
I removed all the newly added services, so only the old ones are left. Fixed remaining of your suggestions.

Reply:
I just tested this and the integration can't find apple tvs that have upgraded to TvOS 15 as the two original services that this was looking for are no longer being announced

Reply:
Listening for `_airplay` without an additional filter is problematic though as it matches way too many things.

Reply:
`{"type":"_airplay._tcp.local.","model":"AppleTV*"}` should pickup AppleTVs


`{"type":"_airplay._tcp.local.","model":"AudioAccessory*"}` should pickup HomePods

Reply:
That should be the most stable AFAICT since apple is least likely to alter the airplay spec as it needs to be interoperable with other vendors

Reply:
Looks like we can only limit by `manufacturer` and not `model` in `zeroconf`

Reply:
..or if we prefer airplay, I'll add support for `model` to `zeroconf`

Reply:
You are right, it won't work. Only the already added services as well as airplay are necessary as neither RAOP nor Companion is currently used by anything (technically RAOP is used by `play_media` but that isn't a problem). We must filter the model as you are suggesting though, only Apple TVs should be found (not HomePods) if we really should stick with just the uplift with no features. Seems unnecessary to spend too much time with this though as I will open a new PR immediately after this being merged, adding the missing services. Processes in all glory, sometimes it's better to just be pragmatic.

Reply:
https://github.com/home-assistant/core/pull/58922

Reply:
@bdraco Once merged, how do I use that? I guess we only want to match "AppleTV*" for now.

Reply:
Right, I see how that works now.